src/pyscipopt/expr.pxi

```diff
@@ -109,6 +109,13 @@ class Term:
     def __repr__(self):
         return 'Term(%s)' % ', '.join([str(v) for v in self.vartuple])
 
+    def _evaluate(self, solution):
```

Comment:
Can you rename `solution` to `value`?

This also applies to later occurrences in this file. In my opinion there is no notion of a `solution` within the context of `Term` and `Expr`.

Reply:
I was gonna suggest to change `value` to `solution` :)

What about using `point`? I guess one evaluates at a `point` to get a `value`

Reply:
Yes, `point` is better.